CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 3.5.0 (2019-12-30)
+
+- FIXED keep order of received dict and it's nested objects
+- Drops python 2 support
```

Comment:
Dropping 2 support should be a v4, not v3.5 thing, mind changing the version number?

Reply:
We should also drop 3.4 support since it's no longer supported.